### PR TITLE
{bio}[intel/2016b] scikit-bio v0.2.3 (WIP)

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/scikit-bio/scikit-bio-0.2.3-intel-2016b-Python-2.7.12.eb [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+#        'source_urls': ['https://pypi.python.org/packages/source/s/setuptools_scm/'],
+#        'source_tmpl': 'setuptools_scm-1.10.1.tar.bz2',
+#    }),
+#    ('simplegeneric', '0.8.1', {
+#        'source_urls': ['https://pypi.python.org/packages/source/s/simplegeneric/'],
+#        'source_tmpl': 'simplegeneric-0.8.1.zip',
+#    }),
+#    ('natsort', '4.0.4', {
+#        'source_urls': ['https://pypi.python.org/packages/source/n/natsort/'],
+#    }),
```
Comment (verdurin) [inline easybuild/easyconfigs/s/scikit-bio/scikit-bio-0.2.3-intel-2016b-Python-2.7.12.eb]: Why are these commented out entries here?

**Comment by verdurin** [comment]
@boegel Sanity check error with `IPython`:
```
Test suite completed for system with the following information:
IPython version: 3.2.3
IPython commit : d296ce3 (installation)
IPython package: /camp/apps/eb/dev/software/IPython/3.2.3-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/IPython
Python version : 2.7.12 (default, Aug  4 2016, 15:30:29) [GCC Intel(R) C++ gcc 5.4 mode]
sys.executable : /camp/apps/eb/software/Python/2.7.12-intel-2016b/bin/python
Platform       : Linux-3.10.0-327.36.2.el7.x86_64-x86_64-with-centos-7.2.1511-Core

Tools and libraries available at test time:
   curses jinja2 jsonschema mistune numpy pexpect pygments requests sqlite3 terminado tornado zmq

Tools and libraries NOT available at test time:
   casperjs matplotlib phantomjs pymongo qt slimerjs sphinx
                                                                                                                                                                                                                   Status: ERROR - 2 out of 13 test groups failed (utils, html). Took 123.256s.

You may wish to rerun these, with:                                                                                                                                                                                   iptest utils html

) (at easybuild/framework/easyblock.py:2066 in _sanity_check_step)
```

**Comment by verdurin** [comment]
Lots of errors like this:
```
======================================================================
ERROR: test suite for <class 'IPython.html.services.contents.tests.test_contents_api.APITest'>
----------------------------------------------------------------------
Traceback (most recent call last):                                                                                                                                                                                   File "/camp/apps/eb/software/Python/2.7.12-intel-2016b/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/suite.py", line 209, in run
    self.setUp()
  File "/camp/apps/eb/software/Python/2.7.12-intel-2016b/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/suite.py", line 292, in setUp
    self.setupContext(ancestor)
  File "/camp/apps/eb/software/Python/2.7.12-intel-2016b/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/suite.py", line 315, in setupContext
    try_run(context, names)
  File "/camp/apps/eb/software/Python/2.7.12-intel-2016b/lib/python2.7/site-packages/nose-1.3.7-py2.7.egg/nose/util.py", line 471, in try_run
    return func()
  File "/camp/apps/eb/dev/software/IPython/3.2.3-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/IPython/html/tests/launchnotebook.py", line 76, in setup_class
    app.initialize(argv=[])
  File "<decorator-gen-113>", line 2, in initialize
  File "/camp/apps/eb/dev/software/IPython/3.2.3-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/IPython/config/application.py", line 75, in catch_config_error
    return method(app, *args, **kwargs)
  File "/camp/apps/eb/dev/software/IPython/3.2.3-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/IPython/html/notebookapp.py", line 1017, in initialize
    self.init_webapp()
  File "/camp/apps/eb/dev/software/IPython/3.2.3-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/IPython/html/notebookapp.py", line 877, in init_webapp
    self.http_server.listen(port, self.ip)
  File "/camp/apps/eb/dev/software/IPython/3.2.3-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/tornado-4.3-py2.7-linux-x86_64.egg/tornado/tcpserver.py", line 126, in listen
    sockets = bind_sockets(port, address=address)
  File "/camp/apps/eb/dev/software/IPython/3.2.3-intel-2016b-Python-2.7.12/lib/python2.7/site-packages/tornado-4.3-py2.7-linux-x86_64.egg/tornado/netutil.py", line 196, in bind_sockets
    sock.bind(sockaddr)
  File "/camp/apps/eb/software/Python/2.7.12-intel-2016b/lib/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 99] Cannot assign requested address
-------------------- >> begin captured logging << --------------------
NotebookApp: INFO: Created profile dir: u'/home/camp/huffmaa/tmp/eb-2lxCj4/tmp1YMv_m/tmpxCgLzn/profile_default'
NotebookApp: INFO: Created profile dir: u'/home/camp/huffmaa/tmp/eb-2lxCj4/tmp1YMv_m/tmpTpwHgb/profile_default'
NotebookApp: INFO: Created profile dir: u'/home/camp/huffmaa/tmp/eb-2lxCj4/tmp1YMv_m/tmpJiMDDA/profile_default'
--------------------- >> end captured logging << ---------------------
```

**Comment by boegel** [comment]
Ancient version, no longer relevant

**Pull request closed without merging**